backoffice/src/Controllers/EmailController.cs

```diff
@@ -50,16 +50,30 @@ public async Task<IActionResult> Compose(Guid id)
     [HttpGet]
     public async Task<IActionResult> SendEmail(Guid id)
     {
-        await _service.SendEmailAsync(id);
+        try
+        {
+            await _service.SendEmailAsync(id);
+            TempData["SuccessMessage"] = "Email sent successfully";
+        }
+        catch (Exception e)
+        {
+            _logger.LogError(e, "Email not found");
+            TempData["NotFoundMessage"] = "Email not found";

```

Comment:
Keep it genetic, so in the views we can have TempData["ErrorMessage"] or something across all the views. Then the error message changes, but the model stays the same

Reply:
Sorted